apps/native/app/src/screens/home/home.tsx

```diff
@@ -93,6 +94,7 @@ export const MainHomeScreen: NavigationFunctionComponent = ({
 }) => {
   useNavigationOptions(componentId)
 
+  useAndroidNotificationPermission()
```

Comment:
~More clear naming would IMHO be something that involves _get_ or _check_ in since we are checking for and getting the permission. If other functions regarding getting permissions have this naming convention then feel free to ignore this, just a bit unclear seeing it here what it does

Reply:
yes that is true I though about it but felt like useCheckAndRequestAndroidNot.... would be to long. Besides hovering over the hook the developer can see a clear description which is commented above the hook